spec/mailers/user_mailer_spec.rb

```diff
@@ -49,6 +50,19 @@
       expect(ActionMailer::Base.deliveries.count).to eq(1)
     end
 
+    it 'logs if the env is set' do
```

Comment:
these are AWESOME, great work! I'll just test this locally and we should be good to go :smile: